bot.go

```diff
@@ -56,6 +59,13 @@ func (x *opBot) Run() {
 			}
 
 		case update.Message != nil:
+			// Stats.
+			if update.Message.From != nil && update.Message.From.UserName == osProgramadoresUserName {
```

Comment:
Why do we only act if the message comes from the bot username? Maybe add an explanation in a comment?